### ISSUE TITLE
There is a very serious bug

### ISSUE THREAD

**Issue by learner-ing** [sgaunet/ssftp#1]
When I use this command "ssftp.exe -i id_rsa C:/test/dir/ kali@192.168.13.131:/tmp/file/" ，Instead of uploading the folder to /tmp/，many files are created in the user directory。
![image](https://user-images.githubusercontent.com/58681115/160086209-dfbd8939-834f-45d2-9e64-eb494cf150a5.png)

**Comment by sgaunet** [comment]
Indeed, never tested on windows in fact. Yes it does not work on windows actually, a library converts automatically the path separator so it's the mess between the client in windows and the server in Linux.
Actually I don't work a lot on this program, I will see if I can do something soon.

**Comment by learner-ing** [comment]
Thank you. I think I may need to refer to this project and write a program myself to realize the program that can be used on windows and Linux.

**Comment by sgaunet** [comment]
I fix the problem, could you check the 0.1-beta5 ?
I need to refactor code before to release the 0.1, it's quite ugly actually.

**Comment by learner-ing** [comment]
When I use this command "ssftp.exe -i id_rsa C:\test.txt kali@192.168.192.142:/tmp/", it tells me that authentication failed. But in fact, my key is right.

**Comment by learner-ing** [comment]
When I use this command "ssftp.exe -i id_rsa C:\test.txt kali@192.168.192.142:/tmp/", it tells me that authentication failed. But in fact, my key is right.
![image](https://user-images.githubusercontent.com/58681115/160218105-45cdea21-4593-4a45-a89a-3d3657fd1628.png)

**Comment by learner-ing** [comment]
I wrote a program to upload files through NC, which may be safer.
